generators/app/templates/src/app/_app.module.ts

```diff
@@ -69,6 +77,16 @@ import { AppRoutingModule } from './app-routing.module';
   ],
   declarations: [AppComponent],
   providers: [
+    {
```

Comment:
Path is the default here, so nothing to add for this use case.
Also this should only be added when not using ionic, as it performs its own override, see ^

Reply:
> Path is the default here, so nothing to add for this use case.

The same to above.

>  Also this should only be added when not using ionic, as it performs its own override,
En. You are right.